assets/js/admin/meta-boxes-order.js

```diff
@@ -39,10 +39,11 @@ jQuery( function ( $ ) {
 			var $this = $( this ),
 				country = $this.val(),
 				$state = $this.parents( 'div.edit_address' ).find( ':input.js_field-state' ),
+				$stateValue = $state.val(),
```

Comment:
variable should not use dollar sign; vars prefixed with $ indicate jQuery objects